turbo.json

```diff
@@ -1,27 +1,19 @@
 {
   "pipeline": {
     "build": {
-      "dependsOn": [
-        "^build"
-      ],
-      "outputs": [
-        "dist/**"
-      ]
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**"]
     },
     "lint": {
       "outputs": []
     },
     "tsc": {
-      "cache": false,
-      "dependsOn": [
-        "^build"
-      ]
+      "outputs": [],
+      "dependsOn": ["^build"]
```

Comment:
I didn't understand the reason to remove the cache, if files didn't change can't we use a cache hit?

Reply:
@danielweinmann